templates/security/compliance-automation.html

```diff
@@ -12,302 +16,534 @@
-    {% include "shared/contextual_footers/_contextual_footer.html" %}
-  {% endwith %}
+  <section class="p-section">
+    {%- call(slot) vf_tiered_list(is_description_full_width_on_desktop=false, is_list_full_width_on_tablet=true) -%}
+      {%- if slot == 'title' -%}
+        <h2>Frequently asked questions about security certifications</h2>
+      {%- endif -%}
+
+      {%- if slot == 'list_item_title_1' -%}
+        <h3 class="p-heading--5">How do I harden my Ubuntu system?</h3>
+      {%- endif -%}
+
+      {%- if slot == 'list_item_description_1' -%}
+        <p>
+          Hardening always involves a tradeoff with usability and performance. The default configuration of Ubuntu LTS releases, as provided by Canonical, balances between usability, performance, and security. However, systems with a dedicated workload are well positioned to benefit from hardening. You can reduce your workload's attack surface by applying an Industry-accepted baseline. At Canonical we recommend applying <a href="/security/cis">the Center for Internet Security (CIS) benchmarks</a> for hardening the configuration of Ubuntu.
+        </p>
+      {%- endif -%}
+
+      {%- if slot == 'list_item_title_2' -%}
+        <h3 class="p-heading--5">How do I comply with PCI-DSS?</h3>
+      {%- endif -%}
+
+      {%- if slot == 'list_item_description_2' -%}
+        <p>
+          PCI-DSS is a payment industry standard and any company that stores, processes, or transmits payment card or cardholder information is required to comply with it. The standard is defined by the Payment Card Industry council and defines measures and processes to secure online financial transactions. The standard is about making business as usual processes like monitoring of security controls, timely response, review of environmental and organizational changes, as well as review of hardware and software being under support by its vendors. For companies with large volumes of transactions compliance with the standard is enforced by an audit of a Qualified Security Assessor (QSA).
+        </p>
+        <p>
+          Achieving and maintaining compliance is a complex and costly process that involves business processes in addition to software requirements. Ubuntu by Canonical contains software and security controls, such as disk encryption, password settings configuration, <a href="/security/fips">cryptographic compliance with FIPS140-2</a>, <a href="/security/cis">CIS hardening</a>, as well as <a href="/pro">a comprehensive Enterprise software maintenance program</a>, to achieve and maintain compliance with the standard.
+        </p>
+      {%- endif -%}
+
+      {%- if slot == 'cta' -%}
+        <a href="/security/contact-us" class="p-button js-invoke-modal">Contact us</a>
+      {%- endif -%}
+    {%- endcall -%}
+  </section>
+
+  <hr class="p-rule is-fixed-width" />
+  <section class="p-strip is-deep">
+    <div class="u-fixed-width u-vertically-center">
```

Comment:
The strip padding itself makes the text look vertically centered so we can remove this 
```suggestion
    <div class="u-fixed-width">
```